types/lib/model.d.ts

```diff
@@ -685,28 +683,28 @@ export interface Hookable {
 /**
  * Options for Model.findOrCreate method
  */
-export interface FindOrCreateOptions extends Filterable, Logging, Transactionable {
+export interface FindOrCreateOptions<TAttributes, TCreationAttributes> extends FindOptions<TAttributes> {
```

Comment:
I made this code just extend FindOptions as other options (like `attributes`) actually are accepted too, per the code inside `model.js` which just directly does a `findOne` with the options.